lib/analyzer/os-release/static.ts

```diff
@@ -18,7 +18,14 @@ export async function detect(
     getOsRelease(extractedLayers, OsReleaseFilePath.Linux),
   );
 
-  // First generic fallback
+  // Generic fallback
```

Comment:
both generic fallbacks? maybe edit the comments to differentiate between both? (l21, l28)